Bottom: 1461c6c13218e99d39868fc8b883c58a8fcaf53f
Top:    ba0de2c90e4e6a80eb0a35da91c5ce216592dd5c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-23 18:30:49 -0400

Refresh of devicetree

---

diff --git a/arch/arm/mach-lpc31xx/board-ea313x.c b/arch/arm/mach-lpc31xx/board-ea313x.c
index dc0c53d..6673805 100644
--- a/arch/arm/mach-lpc31xx/board-ea313x.c
+++ b/arch/arm/mach-lpc31xx/board-ea313x.c
@@ -34,18 +34,16 @@
 #include <linux/gpio.h>
 #include <linux/of_platform.h>
 
-
 #include <asm/system.h>
-#include <mach/hardware.h>
 #include <asm/irq.h>
 #include <asm/pgtable.h>
 #include <asm/page.h>
 #include <asm/sizes.h>
-
 #include <asm/mach/map.h>
 #include <asm/mach-types.h>
-
 #include <asm/mach/arch.h>
+
+#include <mach/hardware.h>
 #include <mach/gpio.h>
 #include <mach/i2c.h>
 #include <mach/board.h>
diff --git a/arch/arm/mach-lpc31xx/dt.c b/arch/arm/mach-lpc31xx/dt.c
index f4606bc..add35ad 100644
--- a/arch/arm/mach-lpc31xx/dt.c
+++ b/arch/arm/mach-lpc31xx/dt.c
@@ -21,10 +21,11 @@
 #include <linux/init.h>
 #include <linux/of_irq.h>
 #include <linux/of_platform.h>
-#include <asm/hardware/vic.h>
+
 #include <asm/mach-types.h>
 #include <asm/mach/arch.h>
 #include <asm/mach/time.h>
+#include <asm/pgtable.h>
 
 #include <mach/hardware.h>
 #include <mach/board.h>
